autoload/test/javascript/jest.vim

```diff
@@ -1,9 +1,10 @@
 if !exists('g:test#javascript#jest#file_pattern')
-  let g:test#javascript#jest#file_pattern = '\v(__tests__/.*|\.(test|spec))\.(js|jsx|coffee)$'
+  let g:test#javascript#jest#file_pattern = '(\v__tests__/.*.(js|jsx|coffee)|(\v.*(spec|test)\.(js|jsx|coffee))$'
```

Comment:
I think this way `(` characters are taken literally, because `\v` is applied only after, the following regex is in my opinion better and shorter:

```
\v(__tests__/.*|(spec|test))\.(js|jsx|coffee)
```